### PR TITLE
Create rui-spinner Component

### PR DESCRIPTION
**Pull request by zulaica**
- Generate `rui-spinner` component.
- Add `rui-spinner` documentation.
- Remove `spin` attribute from `rui-icon`.
- Update `rui-icon` documentation.

### CODE REVIEW

File: addon/components/rui-spinner/component.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  classNames: ['fa'],
+  classNameBindings: ['typeComputed'],
+
+  // Defaults
+  type: 'spinner',
+
+  // Constructors
+  classPrefix: 'fa',
+  // NOTE: Supported spinner icons.
+  types: ['circle-o-notch', 'cog', 'gear', 'refresh', 'spinner'],
```
Comment (kscherling) [inline addon/components/rui-spinner/component.js]: We will want to control what the global spinner style looks like so no need to have user defined options.  Let's default to the 'circle-o-notch' for now (unless you like something better).

Comment (kscherling) [inline addon/components/rui-spinner/component.js]: We will want to be able to effect the size as well.  I think a default one and a 'lg' size is probably sufficient.

Comment (zulaica) [inline addon/components/rui-spinner/component.js]: `circle-o-notch` works for me. Agreed on size control with the default set to `lg`.

Comment (kscherling) [inline addon/components/rui-spinner/component.js]: Cool. Actually default can be the current size that should fit inline with text elements.  lg should be optional for like page loading, etc.

Comment (zulaica) [inline addon/components/rui-spinner/component.js]: Yeah, I misread your comment. Default standard, lg optional. Wrapping this up.

File: addon/components/rui-spinner/component.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    if (!findSize) {
+      return null;
+    }
+
+    if (sizes.indexOf(findSize) === -1) {
+      Ember.Logger.warn('rui-spinner: You specified an unsupported \'size\' property, so we\'ve set it to the standard size. Please either omit the attribute for standard size or choose from one of the following: [\'lg\'].');
+      return null;
+    }
+
+    return 'rui-spinner-' + resolvedSize;
```
Comment (kscherling) [inline addon/components/rui-spinner/component.js]: take a look here https://fortawesome.github.io/Font-Awesome/examples/#larger

**Pull request merged**